docs/streams/developer-guide/config-streams.html

```diff
@@ -461,9 +461,9 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
             <td>86400000 milliseconds (1 day)</td>
           </tr>
           <tr class="row-even"><td>window.size.ms</td>
```

Comment:
I believe in your other PR, you added `(Deprecated)` here?

Let's be consistent.

Reply:
About the other config. If it's missing, we should actually add a new row for it

Reply:
will need to do a rebase/merge to get it in. wanted to make sure things look good so far before adding the dep'd notice